config/checkstyle/checkstyle.xml

```diff
@@ -180,10 +183,14 @@
     <module name="VariableDeclarationUsageDistance"/>
     <module name="CustomImportOrder">
       <property name="sortImportsInGroupAlphabetically" value="true"/>
-      <property name="separateLineBetweenGroups" value="true"/>
-      <property name="customImportOrderRules" value="STATIC###THIRD_PARTY_PACKAGE"/>
+      <property name="separateLineBetweenGroups" value="false"/>
```

Comment:
Is that consistent with google style for java?

Reply:
they have it set to true